### ISSUE TITLE
fix account-detail and delegate-detail after @gny/client upgrade to 1.0.106

### ISSUE THREAD

**Issue by a1300** [GNYIO/gny-explorer#148]
The API endpoint `/api/accounts?address=` with the `address` query parameter changed

from:
```json
{
  "success": true,
  "account": {
    "address": "G44EfJSK6tcwr7XG9aZGZz2byAztV",
    "balance": "92152540",
    "secondPublicKey": null,
    "lockHeight": "0",
    "lockAmount": "0",
    "isDelegate": 0,
    "username": null,
    "publicKey": null
  },
  "latestBlock": {
    "height": "3489955",
    "timestamp": 106579180
  },
  "version": {
    "version": "1.0.93",
    "build": "Mon Apr 04 2022 15:27:00 GMT+0000 (Coordinated Universal Time)",
    "net": "mainnet"
  }
}
```

to:
```json
{
  "success": true,
  "address": "G44EfJSK6tcwr7XG9aZGZz2byAztV",
  "username": null,
  "gny": "23248736039641",
  "publicKey": null,
  "secondPublicKey": "ab5099b82ed7d16e283b5cb7c837de24c7c61b9286b69c5277bacb2bff5dd078",
  "isDelegate": 1,
  "isLocked": 1,
  "lockHeight": "200001",
  "lockAmount": "10000000000",
  "_version_": 20140
}
```

**Comment by a1300** [comment]
This broke the `/account-detail` and `/delegate-detail` page

**Comment by a1300** [comment]
Fixed with PR: #149 and #150
